### PR TITLE
chore: update cloudformation name during remix init

### PR DESCRIPTION
**Pull request by mcansh**
replaces the CloudFormation name during remix.init

Signed-off-by: Logan McAnsh <logan@mcan.sh>

### CODE REVIEW

**Comment by tbeseda** [comment]
btw the --production CloudFormation stack name will be like `RemixGrungeStackA492Production`

File: remix.init/index.js [hunk 1]
```diff
@@ -14,15 +15,18 @@ async function main({ rootDirectory }) {
   const EXAMPLE_ENV_PATH = path.join(rootDirectory, ".env.example");
   const ENV_PATH = path.join(rootDirectory, ".env");
   const PACKAGE_JSON_PATH = path.join(rootDirectory, "package.json");
+  const README_PATH = path.join(rootDirectory, "README.md");
 
   const DIR_NAME = path.basename(rootDirectory);
   const SUFFIX = getRandomString(2);
   const APP_NAME = DIR_NAME + "-" + SUFFIX;
+  const CLOUDFORMATION_NAME = toLogicalID(APP_NAME)
```
Comment (MichaelDeBoey) [inline remix.init/index.js:23]:
```suggestion
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: remix.init/index.js (hunk 1 above)
Comment (mcansh) [reply remix.init/index.js:23]: hey yep, i was going to use it and then didn't 🙈

**Review by kentcdodds — APPROVED** [review]
Oh nice!

**Pull request merged**